source/more-info/unsupported/virtualization_image.markdown

```diff
@@ -0,0 +1,16 @@
+---
+title: "Virtualized-Image"
```

Comment:
```suggestion
title: "Virtualization in use"
```

Reply:
Doesn't that title imply that virtualization isn't supported? I see that I can uses spaces here, I thought I couldn't for some reason. But I'm not sure I agree with that title. What about just "Virtualized image"?

Reply:
Hm, yeah good point. Maybe:

```suggestion
title: "Image not suited for virtualization"
```

Reply:
After chatting we settled on "Incorrect image for virtualization"